README.md

```diff
@@ -677,6 +691,22 @@ The returned setter function accepts an array of workbook selection elements
 function setVariableCallback(value: WorkbookSelection[]): void;
 ```
 
+#### useActionTrigger()
+
+Returns a callback function for the registered action trigger to trigger the action(s).
+
+```ts
+function useActionTrigger(triggerId: string);
+```
+
+Arguments
+
+- `triggerId : string` - The ID of the action trigger
```

Comment:
```suggestion
- `triggerId: string` - The ID of the action trigger
```

Reply:
@Pearce-Ropion Is there a specific reason why there's an extra space before the colon for all other fields introduced in this readme? Was trying to keep my change consistent with the rest of the doc and don't mind reformatting them if necessary.

Reply:
Um probably to make it easier to read. I think leave it as is honestly